src/sdk/cpp/lib/CMakeLists.txt

```diff
@@ -34,6 +34,8 @@ endif()
 set (THREADS_PREFER_PTHREAD_FLAG ON)
 find_package(Threads REQUIRED)
 
+find_library(LIBDL dl)
```

Comment:
Does this imply that we are 'limited' to dynamic linking?

Reply:
Yes